board_touch:
  board_dimensions:
    width: 0.16
    length: 0.26
    height: 0.07
  clearance: 0.07
  force_threshold: 7

stiffness:
  K_pos: 600
  K_z: 1000
  K_ori: 50

pose_box_ref_guess:
  trans_ref_guess: [0.51, 0.137, 0.1]
  rot_ref_guess: [0.7148849557796, 0, 0, -0.699187185237258]

transform_cam:
  trans_cam: [0.483, 0.021, 0.58]
  rot_cam: [0.006, 0.734, -0.679, 0.006]

cartesian_pose_topic_name: '/cartesian_pose'
force_torque_ext_topic: '/force_torque_ext'
equilibrium_pose_topic: '/equilibrium_pose'
pose_ref_2_new_topic: '/pose_ref_2_new'
